src/scripts/components/header/header.styl

```diff
@@ -24,5 +24,14 @@
   white-space: nowrap
   font-size: emCalc(18px)
 
-.empty
+.playButton
```

Comment:
Please use a more general name here. The whole purpose of the children here is that it doesn't matter what you render inside, so it could also be something other then a play button